.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Why is this vscode file here? Is it something local to you and it was added accidentally?

Reply:
it was added accidentally